gui_views/catalog_view.py

```diff
@@ -53,18 +53,57 @@ def createSymObject(self, item):
                 pass
             return
 
+        new_parent = None
+
+        if len(self.state.selected_sym_objects) == 1:
+            new_parent = self.state.selected_sym_objects[0]
+
         del self.state.selected_sym_objects[:]
         #modify state to accomodate the new object
-        self.state.selected_sym_objects[0] = \
+        new_object = \
             self.state.scene.addObjectToScene("component", item.text(0), name)
-        self.state.selected_sym_objects[0].instance_params = \
+        new_object.instance_params = \
             copy.deepcopy(self.catalog[item.parent().text(0)][item.text(0)]['params'])
-        self.state.selected_sym_objects[0].instance_ports = \
+        new_object.instance_ports = \
             copy.deepcopy(self.catalog[item.parent().text(0)][item.text(0)]['ports'])
-        self.state.selected_sym_objects[0].initPorts()
+        new_object.initPorts()
```

Comment:
overwriting rohit's changes?

Reply:
Yes, spoke with Rohit and we agreed that we don't want to highlight objects on load by default - only if we are adding a child through the catalog (which we do later on in the function).